runelite-client/src/test/java/net/runelite/client/util/TextTest.java

```diff
@@ -42,4 +42,16 @@ public void removeTags()
 		assertEquals("Remove no tags", Text.removeTags("Remove no tags"));
 	}
 
+	@Test
+	public void unescapeTags()
+	{
+		assertEquals("Test", Text.unescapeTags("Test"));
+		assertEquals("<:)", Text.unescapeTags("<lt>:)"));
+		assertEquals(">:)", Text.unescapeTags("<gt>:)"));
+		assertEquals("\n", Text.unescapeTags("<br>"));
+		assertEquals("<col=FFFFFF This is a very special message.</col> >:)", Text.unescapeTags("<col=FFFFFF This is a very special message.</col> <gt>:)"));
+		assertEquals("<col=FFFFFF This is a very special message.</col> <:)", Text.unescapeTags("<col=FFFFFF This is a very special message.</col> <lt>:)"));
+		assertEquals("<col=FFFFFF This is a very special message.</col> \n:)", Text.unescapeTags("<col=FFFFFF This is a very special message.</col> <br>:)"));
+		assertEquals("<col=FFFFFF This is a very special message.</col>Not so much.", Text.unescapeTags("<col=FFFFFF This is a very special message.</col>Not so much."));
```

Comment:
I meant test a string with some tag present and observe that the tag would be preserved in the result. eg. `<col=FFFFFF>Colored text</col> and some other text`

Reply:
My bad, I didn't realize what I had added there.